config/config.exs

```diff
@@ -19,4 +19,22 @@ config :appsignal, :config,
   active: false,
   name: "Bullhorn"
 
+config :bullhorn, Bullorn.Mailer, adapter: Bamboo.LocalAdapter
+
+config :bullhorn,
```

Comment:
This seems like it should be code instead of config. You're never going to (be able to) change this without having to make code changes to support it.

Reply:
I'd like to leave this as configuration even if it requires a new deploy to change since the values are themselves configuration for how we handle messages.